File: app/src/main/java/com/github/gotify/messages/provider/MessageState.java
package com.github.gotify.messages.provider;

import com.github.gotify.client.model.Message;
import java.util.ArrayList;
import java.util.List;

public class MessageState {
    public static final long ALL_MESSAGES = -1;

    long appId;
    boolean loaded;
    boolean hasNext;
    long nextSince = 0;
    List<Message> messages = new ArrayList<>();
}

File: app/src/main/java/com/github/gotify/messages/provider/MessageState.kt
package com.github.gotify.messages.provider

import com.github.gotify.client.model.Message

internal class MessageState {
    var appId = 0L
    var loaded = false
    var hasNext = false
    var nextSince = 0L
    var messages = mutableListOf<Message>()

    companion object {
        const val ALL_MESSAGES = -1L
    }
}
